about.html

```diff
@@ -5,64 +5,82 @@
+          <li><a href="#">Products</a></li>
+          <li><a href="#">Blog</a></li>
+          <li><a href="#">Contact</a></li>
+      </ul>
+  </nav>
+  </div>
+</header>
+
 <body>
+  
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
+      <img src="img/jumbo-about.png">
     
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+    <section class="about">
+      <h6>About</h6>
+
+      <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. 
+      Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. 
+      Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+    </section>
+
+  <div class="abt-container">
+    <section>
+        <img src="img/about-plan.png" alt="strategy">
+        <h6>Strategy</h6>
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+        <button type="button">Learn More</button>
+      </section>
+
+      <section>
+        <img src="img/about-working.png" alt="strategy">
+        <h6>How We Work</h6>
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+        <button type="button">Learn More</button>
+  </section>
+  </div>
+
+  <div class="abt-container2">
+    <section>
+      <img src="img/about-office.png" alt="strategy">
```

Comment:
You have the right idea but maybe forgot about it, the alt attribute in each img tag is set to "strategy". For accessibility purposes, it's recommended that the attribute reflect the sections more accurately.